pkg/services/accesscontrol/ossaccesscontrol/ossaccesscontrol.go

```diff
@@ -54,9 +54,17 @@ func (ac *OSSAccessControlService) getUsageMetrics() interface{} {
 	return 1
 }
 
-// Evaluate evaluates access to the given resource
-func (ac *OSSAccessControlService) Evaluate(ctx context.Context, user *models.SignedInUser, permission string, scope ...string) (bool, error) {
-	return evaluator.Evaluate(ctx, ac, user, permission, scope...)
+// Evaluate evaluates access to the given resources
+func (ac *OSSAccessControlService) Evaluate(ctx context.Context, user *models.SignedInUser, evaluator eval.Evaluator) (bool, error) {
+	timer := prometheus.NewTimer(metrics.MAccessEvaluationsSummary)
+	defer timer.ObserveDuration()
+	metrics.MAccessEvaluationCount.Inc()
+
+	permissions, err := ac.GetUserPermissions(ctx, user)
+	if err != nil {
+		return false, err
+	}
+	return evaluator.Evaluate(accesscontrol.GroupPermissions(permissions))
```

Comment:
I think the previous usage of evaluator.Evaluate was to avoid  code duplication since the only difference between evaluations (OSS, ENT) is the GetUserPermissions internal.
Now that I think of it, I wonder why we even display the Evaluate function in the AccessControl interface. We don't have to tie it to a specific implementation. I might be missing something there,  WDYT?
@alexanderzobnin  what's your opinion as well?